app/views/layouts/spotlight/spotlight.html.erb

```diff
@@ -25,6 +25,15 @@
     <% end %>
     <%= stylesheet_link_tag "https://www.stanford.edu/su-identity/css/su-identity.css" %>
     <%= javascript_include_tag "application", async: Settings.async_javascript %>
+    <script type="text/javascript">
+     Honeybadger.configure({
+       apiKey: '<%= Honeybadger.config.get(:api_key) %>',
+       environment: '<%= Honeybadger.config.get(:env) %>',
+       debug: true,
```

Comment:
Should we output the debug info here? Has this been discussed already?

Reply:
It hasn't been discussed yet. I think it should probably be turned off. I'll make the change.